flaml/integrations/oai/completion.py

```diff
@@ -390,18 +458,21 @@ def tune(
             optimization_budget (float, optional): The optimization budget.
             num_samples (int, optional): The number of samples to evaluate.
             **config (dict): The search space to update over the default search.
```

Comment:
Does this API allow users to directly a dict of search space with the same format of `default_search_space`?

Reply:
No, it doesn't. Prompt and stop need special processing.